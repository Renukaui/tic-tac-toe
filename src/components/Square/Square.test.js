import React from 'react';
import Square from './Square';
import Adapter from 'enzyme-adapter-react-16';
import {shallow, configure} from 'enzyme';

configure({adapter: new Adapter()});

describe("Square Component", () => {
  let props;
  beforeEach(() => {
    props = {
      history: [{ 
          squares:Array(9).fill(null), 
          clickIndex:null  
      }]
    }
  })

  it('should match snapshot', () => {
    const wrapper = shallow(<Square {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    shallow(<Square {...props}/>);
  });
});