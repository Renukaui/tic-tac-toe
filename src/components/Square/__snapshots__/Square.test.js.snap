// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square Component should match snapshot 1`] = `ShallowWrapper {}`;
